src/components/Title/Title.tsx

```diff
@@ -0,0 +1,15 @@
+import { STitle } from './Title.styles';
+
+interface TitleProps {
+  text: string;
+}
+
+function Title({ text }: TitleProps) {
+  return (
+    <STitle weight={600} fontSize={32}>
```

Comment:
`weight={600} fontSize={32}`
It looks like a styling thing, you can just define it on style level. Do you really expect it to be dynamic props?